lib/node/runner/jsonrpc_test.go

```diff
@@ -0,0 +1,264 @@
+package runner
```

Comment:
Could we have an integration test here ? Testing from Go will not catch changes in the API that are reflected on the client and server.

Reply:
@Geod24 Good point, we can.